overlays/DIMO/dimo.vspec

```diff
@@ -66,3 +66,84 @@ Vehicle.Powertrain.TractionBattery.Charging.AddedEnergy:
   datatype: float
   unit: kWh
   description: Amount of charge added to the high voltage battery during the current charging session, expressed in kilowatt-hours.
+
+Vehicle.Powertrain.TractionBattery.Charging.ChargeCurrent.AC:
+  datatype: float
+  type: sensor
+  unit: A
+  description: Current AC charging current (rms) at inlet. Negative if returning energy to grid.
+
+Vehicle.ADAS.CruiseControl.CruiseFollowTime:
+  datatype: float
+  type: actuator
+  description: Tesla cruise follow distance seems to be directly proportional to time, but is not seconds
+
+Vehicle.MaxSpeedAllowed:
+  datatype: float
+  type: actuator
+  unit: km/h
+  description: Max speed the vehicle can achieve while Speed Limit Mode is active
+
+Vehicle.ADAS.ObstacleDetection.Forward.Setting:
+  datatype: string
+  allowed: ['Average', 'Early', 'Late', 'Default', 'Off']
+  type: actuator
+  description: Forward collision warning settings, possible values are 'Average', 'Early', 'Late', 'Default', and 'Off'
+
+Vehicle.GuestModeEnabled:
```

Comment:
https://support.toyota.com/s/article/What-is-Sign-out-or-Guest-mode

More universal than I thought